src/OpenLoco/openloco.vcxproj

```diff
@@ -62,10 +62,12 @@
     <ClCompile Include="Localisation\Languages.cpp" />
     <ClCompile Include="Localisation\StringManager.cpp" />
     <ClCompile Include="Localisation\Unicode.cpp" />
+    <ClCompile Include="Map\AnimationManager.cpp" />
     <ClCompile Include="Map\MapGenerator.cpp" />
     <ClCompile Include="Map\SurfaceTile.cpp" />
```

Comment:
You've removed this file, right?

Reply:
No I'm about to readd it for a tile update so didn't see the point